tools/universe/package_builder.py

```diff
@@ -18,7 +18,7 @@
 
 _jre_url = "https://downloads.mesosphere.com/java/server-jre-8u192-linux-x64.tar.gz"
 _libmesos_bundle_url = (
-    "https://downloads.mesosphere.com/libmesos-bundle/libmesos-bundle-1.11.0.tar.gz"
+    "https://downloads.mesosphere.com/libmesos-bundle/libmesos-bundle-1.12.0.tar.gz"
```

Comment:
looks like this branch isn't rebased on master properly, is it? This change was introduced in https://github.com/mesosphere/dcos-commons/pull/2832